apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/CacheStore.java

```diff
@@ -34,4 +34,6 @@
     return aggregatedDependentKeys;
   }
 
+  public abstract void clearAll();
```

Comment:
We should not forget to add javadoc and mention that this function shouldn't be called directly from client only via Cache.clearAll